src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <ul class = "stopwatch" >
```

Comment:
You need to remove the spaces around the equals sign, it will make your code more readable

Reply:
Check other similar cases.